Makefile

```diff
@@ -199,6 +199,11 @@ endif
 	mv config/base/config.properties.bak config/base/config.properties
 	mv config/base/manager_image_patch.yaml.bak config/base/manager_image_patch.yaml
 
+### install_plugin_templates: Deploy sample plugin templates to namespace devworkspace-plugins:
+install_plugin_templates: _print_vars
+	$(K8S_CLI) create namespace devworkspace-plugins || true
+	$(K8S_CLI) apply -f samples/plugins -n devworkspace-plugins
```

Comment:
I suppose we need to grant all users read-only access DevWorkspaceTemplates in this namespace.
And probably even all ServiceAccounts, otherwise how theia or dashboard will list them.

Reply:
I see RBAC for SAs are updated, but usual users did not get such permissions yet. Like we have for view and edit for devworkspaces

https://github.com/amisevsk/devworkspace-operator/blob/fcad74e80920ca16b3dd432e280c79db1172b964/config/components/rbac/edit-workspaces-cluster-role.yaml

Reply:
Good catch -- these files are written manually (not generated) so I'll need to add DevWorkspaceTemplates here.

Reply:
Done in 83f5133